parsers/cxx/cxx_parser_internal.h

```diff
@@ -220,7 +220,9 @@ typedef enum _CXXParserKeywordState
 	// "const" has been seen at block level
 	CXXParserKeywordStateSeenConst = (1 << 9),
 	// "volatile" has been seen at block level
-	CXXParserKeywordStateSeenVolatile = (1 << 10)
+	CXXParserKeywordStateSeenVolatile = (1 << 10),
+	// __attribute__((deprecated)) has been seen
+	CXXParserKeywordStateSeenAttributeDeprecated = (1 << 11)
```

Comment:
It's not [legal C89](http://port70.net/~nsz/c/c89/c89-draft.html#3.5.2.2) (I assume uctags uses C89 since it doesn't use `AC_PROG_C99` in `configure.ac` to get C99 features). Kind of pedantic though, just sayin :)